bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RetryingReadRowsOperation.java

```diff
@@ -307,12 +301,14 @@ int getTimeoutRetryCount() {
   }
 
   @VisibleForTesting
-  BackOff getCurrentBackoff() {
-    return currentBackoff;
+  RowMerger getRowMerger() {
+    return rowMerger;
   }
 
+
   @VisibleForTesting
-  RowMerger getRowMerger() {
-    return rowMerger;
+  boolean performingRetries() {
```

Comment:
Is this method necessary? can't the test just use the existing inRetryMode?

Reply:
I made `inRetryMode()` to be able to use it in the scanner package.